app/scripts/Annotations2dTrack.js

```diff
@@ -22,7 +23,7 @@ class Annotations2dTrack extends TiledPixiTrack {
 
     this.options.minSquareSize = +this.options.minSquareSize;
 
-    const { publish, subscribe, unsubscribe } = create({});
+    const { publish, subscribe, unsubscribe } = createPubSub({});
```

Comment:
`createPubSub` always creates a new pubSub instance. When you take a look here: https://github.com/flekschas/pub-sub/blob/master/src/index.js#L66 you'll see that `createPubSub` will create a new stack given the object you pass in or otherwise will create a new stack object by default. Having said that, I just realized that `createPubSub()` should be called without `{}`. Will fix that.